.github/workflows/integration_tests.yaml

```diff
@@ -0,0 +1,43 @@
+name: Run Integration Tests
+
+on:
+  pull_request:
+    types: [assigned, opened, synchronize, reopened]
+    paths-ignore:
+      - "README.md"
+  
+  workflow_dispatch:
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.9"
+
+      - name: Setup python
+        run: python -m venv .venv
```

Comment:
I think this doesn't do anything because we never activate the virtual environment

Reply:
yes, you're right! i removed the virtual environment parts in https://github.com/fal-ai/fal/pull/22/commits/61217f3da539d96ab52742c090b48da920919c6c, but it seems like I forgot this section

Reply:
https://github.com/fal-ai/fal/pull/25